docs/configuration.md

```diff
@@ -118,3 +118,93 @@ _Default Value_
 _Runtime Configurability_
 
 Supported
+
+## DownloadDeps
+The **DownloadDeps** configuration option controls whether or not RedisGears will attempt to download missing Python dependencies.
+
+_Expected Value_
+
+0 (disabled) or 1 (enabled)
+
+_Default Value_
+
+"1"
+
+_Runtime Configurability_
+
+Not Supported
+
+## DependenciesUrl
+The **DependenciesUrl** configuration option controls the location from which RedisGears tries to download its Python dependencies.
+
+_Expected Value_
+
+URL-like string
+
+_Default Value_
+
+The default value is specific to the RedisGears version.
+
+_Runtime Configurability_
+
+Not Supported
+
+## DependenciesSha256
+The **DependenciesSha256** configuration option specifies the SHA265 hash value of the Python dependencies. This value is verified after the dependencies have been downloaded and will stop the server's startup in case of a mismatch.
+
+_Expected Value_
+
+String
+
+_Default Value_
+
+The default value is specific to the RedisGears version.
+
+_Runtime Configurability_
+
+Not Supported
+
+## PythonInstallationDir
+The **PythonInstallationDir** configuration option specifies the path for RedisGears' Python dependencies.
+
+_Expected Value_
+
+String
+
+_Default Value_
+
+?
```

Comment:
@MeirShpilraien this needs to be defined

Reply:
Sets to : /var/opt/redislabs/modules/rg